API-reference.md

```diff
@@ -414,7 +414,7 @@ Sends telemetry to the endpoint.
 
 ### addTelemetryInitializer
 
-        public addTelemetryInitializer(telemetryInitializer: (envelope: Telemetry.Common.Envelope) => boolean)
+        public addTelemetryInitializer(telemetryInitializer: (envelope: Telemetry.Common.Envelope) => boolean | void)
```

Comment:
Yes, it's a back compatibility change. We were always accepting bool|void, but definition types only allowed boolean.